NDcPP_v3_0.adoc

```diff
@@ -3049,7 +3049,7 @@ _Although this is out-of-scope of this cPP, it is strongly advised to protect al
 
 *FPT_TUD_EXT.2.1* The TSF shall check the validity of the code signing certificate before installing each update.
 
-*FPT_TUD_EXT.2.2* *If revocation information is not available for a certificate in the trust chain that is not a trusted certificate designated as a trust anchor, the TSF shall [selection: _not install the update, allow the Administrator to choose whether to accept the certificate in these cases_].*
+*FPT_TUD_EXT.2.2* If revocation information is not available for a certificate in the trust chain that is not a trusted certificate designated as a trust anchor, the TSF shall [selection: _not install the update, allow the Administrator to choose whether to accept the certificate in these cases_].
```

Comment:
@Kirky-J Bold also needs to be corrected in Section C.4.4.1.

Reply:
Nice catch. Thanks. Fixed.